invenio_oaiharvester/errors.py

```diff
@@ -19,7 +19,44 @@
 
 """OAI harvester errors."""
 
+NAME_OR_URL_ERROR_MSG = """
```

Comment:
Traditionally, we have not been using this technique to have the Exception messages as "constants". The reason being that when reading the code, one would have to open this file to see what the message would say, it is not directly clear when it is being raised. Now, if this text was being repeated a lot there could be an advantage, but I see that they are only used once for now. 

What I am saying is that I would prefer the messages to be put directly as string when raising for now. And you can shorten the messages to one sentence.